docs/equip/stut/stut_equipments.md

```diff
@@ -0,0 +1,52 @@
+# Stut equipments
+
+| 概要 | 部位 | 強化 | 転生 | 潜在(上/下) | メモ/不満点 |
+| :---- | :---- | :---- | :---- | :---- | :---- |
+| ![equipment](./res/stut_equipments_20210822.png "装備") | ヴェラ指★17 | 専用書埋め | - | <span style="color: lime;">24%</span> / なし | 雑魚<br>変えたい |

```

Comment:
The markdown extension is not available on github.
Move picture to out of table and stylize.